### ISSUE TITLE
Screenshots are for another project

### ISSUE THREAD

**Issue by maximilliangeorge** [mably/ln-dice#5]
Hey, this may seem peripheral, but i think the screenshots are of another project? Great work with everything, i really admire your efforts :)

**Comment by mably** [comment]
Definitely.  These are rather old screenshots of the https://github.com/mably/lncli-web project which is included in the ln-dice project.
